_includes/manuals/2.0/1.2_release_notes.md

```diff
@@ -4,6 +4,20 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### New sidekiq-based implementation for Dynflow
```

Comment:
Foreman now uses Dynflow backed by Sidekiq by default on all supported platforms. This change is described in greater detail in [Upcoming changes to Dynflow](https://community.theforeman.org/t/upcoming-changes-to-dynflow/14926). The original Dynflow executor service (called `dynflowd`) was replaced by `redis` and a set of `dynflow-sidekiq@*` services. This new deployment model should address issues we have seen when restarting the Dynflow executor. It also allows us scale more easily, dedicate workers to specific queues and increase the overall throughput.

Reply:
thanks, updated!

Reply:
Sorry for taking so long